scripts/rev_cron.php

```diff
@@ -0,0 +1,197 @@
+
+            //Update the rev status
+            $order->status = $rev_order->getStatus();
+            
+            if ($rev_order->isComplete()) {
+                echo "\t order is complete, fetching attachments..." . PHP_EOL;
+                
+                $attachments = $rev_order->getAttachments();
+                sleep(1); //be nice
+
+                $media_text_track = new UNL_MediaHub_MediaTextTrack();
+                $media_text_track->media_id = $media->id;
+                $media_text_track->source = UNL_MediaHub_MediaTextTrack::SOURCE_REV;
+                $media_text_track->revision_comment = 'Order: ' . $rev_order->getOrderNumber();
+                $media_text_track->save();
+
+                //Mark the order as complete
+                $order->status = UNL_MediaHub_RevOrder::STATUS_MEDIAHUB_COMPLETE;
+                
+                foreach ($attachments as $attachment) {
+                    if (!$attachment->isMedia()) {
+                        //Only save non-media attachments
+                        
+                        echo "\t creating a new text track file" . PHP_EOL;
+                        $media_text_track_file = new UNL_MediaHub_MediaTextTrackFile();
+                        $media_text_track_file = new UNL_MediaHub_MediaTextTrackFile();
+                        $media_text_track_file->media_text_tracks_id = $media_text_track->id;
+                        $media_text_track_file->kind = UNL_MediaHub_MediaTextTrackFile::KIND_CAPTION;
+                        $media_text_track_file->format = UNL_MediaHub_MediaTextTrackFile::FORMAT_VTT;
+                        $media_text_track_file->language = 'en';
+                        $media_text_track_file->file_contents = $attachment->getContent('.vtt');
+                        $media_text_track_file->save();
+                    }
+                    
+                    sleep(1); //be nice
+                }
+
+                //update the media to point to the new text track
+                $media->media_text_tracks_id = $media_text_track->id;
+                $media->dateupdated = date('Y-m-d H:i:s');
+                $media->save();
+                
+                //Get the email for the person who requested the order
+                $data = file_get_contents('http://directory.unl.edu/?uid='.$order->uid.'&format=json');
+                $data = json_decode($data, true);
+                
+                //Send them an email if we can...
+                if (is_array($data) && isset($data['mail'][0])) {
+                    $message = Swift_Message::newInstance();
+                    
+                    $message->setTo($data['mail'][0]);
+
+                    // Give the message a subject
+                    $message->setSubject('MediaHub: Your caption order is complete for: ' . $media->title);
+
+                    // Set the From address with an associative array
+                    $message->setFrom(array('wdn@unl.edu' => 'UNL WDN'));
```

Comment:
Sub-addressing is _much_ better be the responded has less control over that. A subject can be (and often is) mangled by various mail clients the can't figure out "RE:RE:RE:RE:"